docs/_docs/1_4_powered_by.cn.md

```diff
@@ -48,10 +48,20 @@ Hudi在Yotpo有不少用途。首先，在他们的[开源ETL框架](https://git
 
 7. ["Building highly efficient data lakes using Apache Hudi (Incubating)"](https://www.slideshare.net/ChesterChen/sf-big-analytics-20190612-building-highly-efficient-data-lakes-using-apache-hudi) - By Vinoth Chandar 
    June 2019, SF Big Analytics Meetup, San Mateo, CA
-   
+
 8. ["Apache Hudi (Incubating) - The Past, Present and Future Of Efficient Data Lake Architectures"](https://docs.google.com/presentation/d/1FHhsvh70ZP6xXlHdVsAI0g__B_6Mpto5KQFlZ0b8-mM) - By Vinoth Chandar & Balaji Varadarajan
    September 2019, ApacheCon NA 19, Las Vegas, NV, USA
 
+9. ["Insert, upsert, and delete data in Amazon S3 using Amazon EMR"](https://www.portal.reinvent.awsevents.com/connect/sessionDetail.ww?SESSION_ID=98662&csrftkn=YS67-AG7B-QIAV-ZZBK-E6TT-MD4Q-1HEP-747P) - By Paul Codding & Vinoth Chandar
+   December 2019, AWS re:Invent 2019, Las Vegas, NV, USA
```

Comment:
Why here need to break a new line? Can we merge into the previous line?

Reply:
> Why here need to break a new line? Can we merge into the previous line?

Just copied from english version.

Reply:
OK~